hepdata/modules/records/utils/submission.py

```diff
@@ -426,6 +453,8 @@ def process_submission_directory(basepath, submission_file_path, recid,
         # Side effect that reviews will be deleted between uploads.
         cleanup_submission(recid, hepsubmission.version, added_file_names)
 
+        # Counter to store current table number.
```

Comment:
Add a comment: 
This is required because the first table is the main submission. Subsequent tables passed to `process_data_file` need to be appended to the DOI, so we need to keep track of the table number!